### PR TITLE
Convert packaging script to MakD

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**
Also some other minor improvements to make files and build scripts.

### CODE REVIEW

**Comment by mihails-strasuns-sociomantic** [comment]
Red :)

**Comment by leandro-lucarella-sociomantic** [comment]
2nd try

**Comment by leandro-lucarella-sociomantic** [comment]
:green_heart:

File: docker/ci.sh [hunk 1]
```diff
@@ -2,6 +2,6 @@
 set -xe
 
 export F=production
-make all
-make test
-make pkg
+make -r all
```
Comment (mihails-strasuns-sociomantic) [inline docker/ci.sh:5]: Title says `-n`. Whom to trust? :)

**Review by mihails-strasuns-sociomantic — CHANGES_REQUESTED** [review]

File: docker/ci.sh (hunk 1 above)
Comment (leandro-lucarella-sociomantic) [reply docker/ci.sh:5]: Code, my code is always perfect.

**Comment by leandro-lucarella-sociomantic** [comment]
Updated commit message.

**Review by mihails-strasuns-sociomantic — APPROVED** [review]

**Pull request merged**